### ISSUE TITLE
[P3.1][Sourcing][Accessibility][ATL testing][Table][WIP]: sublevels in tree table not being read properly , row number missing 

### ISSUE THREAD

**Issue by I543348** [SAP/fundamental-ngx#10618]
#### Is this a bug, enhancement, or feature request?
Bug

#### Describe your proposal.
sublevels in tree table not being read properly , row number missing 
pls see below screenshot from w3 and NGX table 
<img width="892" alt="access11" src="https://github.com/SAP/fundamental-ngx/assets/88182358/eea6e8e3-7bef-4278-9b7d-251a0ecb0d75">

<img width="808" alt="Screenshot 2023-09-27 at 2 50 00 PM" src="https://github.com/SAP/fundamental-ngx/assets/88182358/f4236bd6-e0bd-4400-9cfc-cf59dd527008">

#### Can you handle that on the application side
No

#### Which versions of Angular and Fundamental Library for Angular are affected? Please, specify the exact version. (If this is a feature request, use current version.)
ng: 15
platform: 0.43.x

#### If this is a bug, please provide steps for reproducing it; the exact components you are using;
platform tree table 

#### In case this is Accessibility related topic, did you consult with an accessibility expert? If not, please do so and share their recommendations. 

reported by ATL team

**Comment by nikolay-kolarov** [comment]
Hello @I543348,

I do not really get the issue. On both of the provided pictures "Row 8" is available as information.
Could you please provide more info on what exactly is missing and with which interaction to access it?

Thanks,
Nikolay

**Comment by I543348** [comment]
@sagar7259 pls add your inputs.
Hi @nikolay-kolarov  this issue was filed by ATL team' testing.

**Comment by sagar7259** [comment]
@nikolay-kolarov the row count is being read correctly but the tree level information for each row is not read for the users and just "1 of 3", "2 of 3" does not make any sense for the users.
 
![image](https://github.com/SAP/fundamental-ngx/assets/143692208/3c5ee70a-6fed-42ba-b14f-203873111075)

please have a look into this attached speech history of the tree level.
Here on traversing into each tree level, its corresponding tree level information ( ex: level 1, level 2) is being read respectively and the same is expected here also.

Regards,
Sagar

**Comment by nikolay-kolarov** [comment]
Hi @sagar7259 ,

Thanks! So now we accept that the rows are announced ok and we need to look into the level? 

Will it be possible to provide a link where I can test it.
I tried the tree here: https://ng-15-downport--fundamental-ngx.netlify.app/#/core/tree and I can hear the level information correclty.

Thanks,
Nikolay

**Comment by sagar7259** [comment]
Hi @nikolay-kolarov ,
The above example provided by you is from normal tree, but the existing issue is from tree table, where both the components are different.

Tree table: https://ng-15-downport--fundamental-ngx.netlify.app/#/platform/table/basic
![image](https://github.com/SAP/fundamental-ngx/assets/143692208/7b6b8bba-de2d-4933-9529-9c7b319286d1)

In this screenshot you can see that there are 2 different levels present for 'Laptop' cell. But JAWS is not reading any level information.

Regards,
Sagar

**Comment by nikolay-kolarov** [comment]
Hello @sagar7259,

Thanks for the details.
I checked further the sample and found something that can be improved in the coding.
There will be a possibility to remove those strange numbers (from your picture: "1 of 5" and "1 of 2").
With this the output will be:
![image](https://github.com/SAP/fundamental-ngx/assets/17043004/c41d6779-43e7-4d91-ab54-51b2b8ae4fb0)

On the image I left on purpose the 1 of 1 on the Benda Laptop (with arrow on the image).

If this is cleared will you be ok with the other output?

I checked on the APG sample here: https://www.w3.org/WAI/ARIA/apg/patterns/treegrid/examples/treegrid-1/?cell=force
and also in UI5 the level is not announced (although properly defined in the HTML): https://ui5.sap.com/#/entity/sap.ui.table.TreeTable/sample/sap.ui.table.sample.TreeTable.JSONTreeBinding


@droshev and team - those counts are coming due to the tabindex="-1" attribute added on all <tr> elements. And this is not needed, as they are not focusable. Still the addition of tabindex -1 forces this count announcement which is confusing. So if sagar confirms that this is the accepted solution for him, you can fix it by removing the tabindex -1 on row level.

Thanks,
Nikolay
